packages/react/src/components/Footer/FooterLogo.js

```diff
@@ -8,6 +8,7 @@
 import React from 'react';
 import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
 import { settings } from 'carbon-components';
+import FooterLogo from '../../../../styles/icons/svg/IBM-8bar-logo--h65-white.svg';
```

Comment:
@kennylam not sure if this import will work since it's pulling in relatively across packages. If for some reason the adopter doesn't pull in the styles package this could break. The reference would have to be:
```
import { FooterLogo } from '@carbon/ibmdotcom-styles/icons/svg/IBM-8bar-logo--h65-white.svg'
```